README.md

```diff
@@ -39,17 +63,16 @@ an Islandora Foundation [Contributor License Agreement][6] or
 * American Philosophical Society
 * common media inc.
 
-## Maintainers
+## Development
 
-* [Daniel Lamb](https://github.com/dannylamb)
+If you would like to contribute, please get involved by attending our weekly [Tech Call](https://github.com/Islandora-CLAW/CLAW/wiki). We love to hear from you!
 
-## License
+If you would like to contribute code to the project, you need to be covered by an Islandora Foundation [Contributor License Agreement](http://islandora.ca/sites/default/files/islandora_cla.pdf) or [Corporate Contributor License Agreement](http://islandora.ca/sites/default/files/islandora_ccla.pdf). Please see the [Contributors](http://islandora.ca/resources/contributors) pages on Islandora.ca for more information.
+
+We recommend using the [claw-playbook](https://github.com/Islandora-Devops/claw-playbook) to get started. If you want to pull down the submodules for development, don't forget to run `git submodule update --init --recursive` after cloning.
```

Comment:
This should be the last lingering CLAW url

Reply:
this should be fixed now